### PR TITLE
Adding diffrence java program

### PR DESCRIPTION
**Pull request by guppi4u**
Hi 
As discussed, added diff program.

### CODE REVIEW

File: Diff.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Diff{
+    public static void main(String[] args) {
+        int x=100;
+        int y=20;
+
+        int diff;
```
Comment (guppikan) [inline Diff.java]: Please follow a variable naming convention.

Comment (guppi4u) [reply Diff.java]: Hi 
Updation done to program

Comment (guppikan) [reply Diff.java]: ok let me check

**Review by guppikan — APPROVED** [review]
Looks ok 
Merging the changes to the main branch

**Comment by guppikan** [comment]
feature2 branch has been merged

**Comment by guppi4u** [comment]
thanks

**Pull request merged**